content/applications/websites/livechat/responses.rst

```diff
@@ -160,24 +152,87 @@ end the conversation.
 Canned responses
 ================
 
-*Canned responses* are customizable inputs where a *shortcut* stands in for a longer response. An
-operator will enter the shortcut, and it will automatically be replaced by the expanded
-*substitution* response in the conversation.
+*Canned responses* are customizable inputs where a typed shortcut populates a longer response. A
+user enters a keyword shortcut, which is then automatically replaced by the expanded substitution
+response. Canned responses save time by allowing users to use shorthand phrases to populate longer
+messages. This also limits the possibility of errors when typing out longer messages because these
+are pre-set messages. This maintains consistency throughout customer interactions.
+
+Canned responses consist of two main components: the *shortcut* and the *substitution*. The shortcut
+is the keyword or key phrase that is to be replaced. The substitution is the longer message that
+replaces the shortcut.
+
+.. _livechat/created-canned-response:
 
 Create canned responses
 -----------------------
 
 To create a new canned response, go to :menuselection:`Live Chat app --> Configuration --> Canned
 Responses --> New`.
 
-From here, type the shortcut command into the :guilabel:`Shortcut` field.
+Type a shortcut command in the :guilabel:`Shortcut` field. Next, click on the
+:guilabel:`Substitution` field, and type the message that will replace the shortcut.
+
+.. tip::
+   Try to connect the shortcut to the topic of the substitution. Not only does this make it easier
+   to use the responses, it prevents the list of responses from becoming disorganized and
+   overwhelming.
+
+In the :guilabel:`Description` field, add any information that provides context for this response,
+such as guidelines for when it should or should not be used.
+
+The :guilabel:`Created by` field automatically populates with the name of the user that creates a
+new response. This field cannot be edited.
+
+To :ref:`share <livechat/sharing-responses>` this response with other users, select one or more
+groups in the :guilabel:`Authorized Group` field that should have access.
```

Comment:
I'm in a 17.0 runbot, and I don't see the Created By field or the Authorized Group field ><
![image](https://github.com/odoo/documentation/assets/23611711/0d71231f-21a7-4292-ae0e-ebed61fa2b37)

If there are additional configurations that need to be made, perhaps include them nearby this paragraph in an important admonition block?

Reply:
As we discussed, I am going to remove these and add it in a later update (these features were added in 17.2, not 17) thank you for catching that!